chpl/chpl-service/src/main/resources/environment.properties.template

```diff
@@ -9,6 +9,7 @@ questionableActivityThresholdDays=0
 summaryEmail=dlucas@ainq.com
 summaryEmailName=summaryStatistics.csv
 summaryEmailPeriodInDays=7
+maxActivityRangeInDays=60

```

Comment:
Stylistically, this properties file is getting hard to read. The section breaks with whitespace are useful